internal/engine/engine.go

```diff
@@ -220,7 +220,7 @@ func (c *craneEngine) ExecuteChecks(ctx context.Context) error {
 		c.results.TestedImage = c.image
 
 		logger.V(log.DBG).Info("running check", "check", check.Name())
-		if check.Metadata().Level == "optional" {
+		if check.Metadata().Level == "optional" || check.Metadata().Level == certcheck.LevelWarn {
```

Comment:
Why not use the Optional const?

Reply:
To keep this PR focused on the required annotations, I was planning a follow up PR to switch all other checks to the constants. I assumed that would be preference then introducing here. But maybe it's fine to just introduce for this one line since it is already changing?